.changeset/forty-onions-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-application-studio': patch
+---
+
+Fix a bug with `PackageableTreeNode` not getting selected for dependency elements.
```

Comment:
Fix bug when opening dependency element does not open corresponding element tree node in the dependency explorer?